myVK/myVK/ViewController/Controller/View/FriendPhotosViewController.swift

```diff
@@ -0,0 +1,110 @@
+// FriendPhotosViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+/// Экран с перелистыванием фото
+final class FriendPhotosViewController: UIViewController {
+    // MARK: - Private Visual Components
+
+    @IBOutlet private var photoImageView: UIImageView!
+
+    // MARK: - Public Properties
+
+    var photoNames: [String] = []
+    var currentPhotoIndex = 0
+
+    // MARK: - Private property
+
+    private var interactiveAnimator: UIViewPropertyAnimator!
+
+    // MARK: - LifeCycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTitle()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        changeColorBars(true)
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        changeColorBars(false)
+    }
+
+    // MARK: - Private Methods
+
+    private func changeColorBars(_ isBlack: Bool) {
+        if isBlack {
```

Comment:
```suggestion
        if isBlack {
```
Замени на тернарный оператор что б не дублировать код, спроси у Еврнеста как это делать